### ISSUE TITLE
Ralentissement

### ISSUE THREAD

**Issue by Fulji** [pie3636/fire-watcher#25]
Enorme ralentissement pour voir les tooltips de l'inventaire.

**Comment by pie3636** [comment]
Est-ce que tu arrives à reproduire le bug dans la nouvelle version?

**Comment by Fulji** [comment]
Oui, suffit de mettre sa souris sur branche, ça affiche la tooltip sans problème. Puis tu monte un petit peu pour qu'elle disparraisse. Enfin tu répètes ces deux étapes rapidement.
Edit : Cela dit, en fait le lag monstrueux était qu'en j'avais une prod variable des singes, ainsi il pouvait y avoir des clics un peu n'improte quand et donc parfois ça laguait beaucoup plus.
